.github/workflows/tests.yaml

```diff
@@ -0,0 +1,52 @@
+name: Tests
```

Comment:
We should come up with some standards for the naming of these workflows. I'm not sure I've seen `tests.yaml` before

Reply:
We did it in the [last one](https://github.com/canonical-web-and-design/canonicalwebteam.discourse-docs/pull/17/files). @jpmartinspt  ask me to use "Tests" for everything, it makes sense because it appears on the PRs later with the "Tests/" prefix, I will update the ones that we did before.